RefactoringPracticeI/GameInventory/Inventory/AgedBrie.cs

```diff
@@ -0,0 +1,12 @@
+﻿
```

Comment:
Subjective: Ideally the code conventions should be applied in every file, for instance, here you start your file with 2 blank lines, next file has 1, and the third one has zero, its really minor, but helps reviewer to focus on actual changes.